src/components/pagination/page/page.jsx

```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import injectSheet from 'react-jss';
 import cn from 'classnames';
+import { Link } from 'react-router-dom';
```

Comment:
This is not a current dependency, is it possible to send it in as a prop instead similar to how `Button` works?

Reply:
agreed, it shouldn't be here